### PR TITLE
Package json lint test

### PR DESCRIPTION
**Pull request by saket2403**
### Summary

Add package json lint test for all packages

Closes #

### Deployment Link

https://terra-clinical-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-clinical-detail-view/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
```
Comment (benbcai) [inline packages/terra-clinical-detail-view/CHANGELOG.md:5]: This should be `Added` for all the changelogs.

Comment (saket2403) [reply packages/terra-clinical-detail-view/CHANGELOG.md]: Done

**Review by benbcai — APPROVED** [review]

**Comment by benbcai** [comment]
@saket2403 Can this be merged or is it blocked?

**Comment by saket2403** [comment]
> @saket2403 Can this be merged or is it blocked?

This PR is ready to be merged @benbcai

**Pull request merged**